tools/test.py

```diff
@@ -260,6 +260,12 @@ def main():
         tmpdir = None
 
     cfg.device = get_device()
+
+    if cfg.device == 'npu':
+        optimiter_config = dict(type='Fp16OptimizerHook', loss_scale='dynamic')
+        cfg.optimizer_config = optimiter_config if \
+            not cfg.optimizer_config else cfg.optimizer_config
```

Comment:
when model testing, there is no runner and no hooks

Reply:
I've revised it, thanks！